File: 1480. Running Sum of 1d Array/runningSumOf1dArrayProgram.java
public class Main {
  public static void main(String[] args) {
    Main m = new Main();
    int[] result = m.runningSum(new int[] {1,1,1,1,1});

    //Java doesn't have foreach loop
    //Java's enhanced for-loop
    //for (Type variable : collection){}
    
    for(int i : result)
    {
    	System.out.println(i);
    }
  }
  
  public int[] runningSum(int[] nums)
  {
  	for(int i = 1; i < nums.length; i++)
    {
    	nums[i] += nums[i-1];
	}
    return nums;
  }
}

File: 1480. Running Sum of 1d Array/runningSumOf1dArrayProgram.cs
using System;

namespace HelloWorld
{
  class Program
  {
    static void Main(string[] args)
    {
    	  Program p = new Program();
        int[] result = p.RunningSum(new int[]{3,1,2,10,1}); 
        foreach(int i in result)
        {
          Console.WriteLine(i);
        }
    }
    
    public int[] RunningSum(int[] nums) {

        for(int i = 1; i < nums.Length; i++ )
        {
            nums[i] += nums[i-1];
        }

        return nums;
    }
  }
}
